src/Sylius/Bundle/CoreBundle/Distributor/TaxesDistributor.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Distributor;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class TaxesDistributor implements TaxesDistributorInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function distribute($taxItems, $taxAmount)
```

Comment:
I find it weird, in real life you're distributing $1000 into 10 parts, the ordering of the arguments seems wrong.